src/chrome/content/rules/OpenCCC.xml

```diff
@@ -0,0 +1,26 @@
+<ruleset name="OpenCCC">
+
+	<target host="openccc.net" />
```

Comment:
Ah, sorry, I had simply missed the exclusion. I agree that this approach is sensible.